design-editor/src/pane/design-editor-element.js

```diff
@@ -294,6 +297,7 @@ class DesignEditor extends DressElement {
 			}
 
 			this._$iframe.one('load', () => {
+				const isDemoVersion = PreferencesManager.getViewState('projectType') === 'demo' ? true : false;
```

Comment:
You can just write: 
`const isDemoVersion = PreferencesManager.getViewState('projectType') === 'demo'`
means the same, but it's shorter